### PR TITLE
capstone: enable clangarm64

### PR DESCRIPTION
**Pull request by heljkon**
Try for clangarm64. Seems to be the last missing dependency for qemu clangarm64

### CODE REVIEW

**Comment by Biswa96** [comment]
Adding mingw_arch does not require pkgrel bump generally.

**Comment by heljkon** [comment]
OK, thanks for this information. Should I reset the pkgrel bump?

BTW: Since there was no clangarm64 check, how can this be tested? I don't own an arm64 machine, so this is only a try...

**Comment by Biswa96** [comment]
> Should I reset the pkgrel bump?

Probably, because there is no change besides mingw_arch.

> I don't own an arm64 machine

We all have an aarch64 machine, don't we? 😉 Though most of them running a _weird_ version of Linux.

> how can this be tested?

For my case, I cross compile packages in x86_64 machine and run it in Windows ARM in Raspberry Pi 4. Others have better hardware. Without arm hardware, Windows ARM can be used in qemu 🐌

**Comment by Biswa96** [comment]
> Has this to be done manually?

IIRC, @jeremyd2019 can test a PR in semi-automatic way.

The program can be cross compiled for Windows ARM without any issue. Do you need any specific test?

**Comment by jeremyd2019** [comment]
I thought about having qemu on arm64, but AFAIK whpx is part of the x86 section of qemu code, so probably wouldn't *just work* on Windows on ARM64.  If the only option is tcg, may as well run tcg on an x86_64 box that is much more common.

I will try this PR though, wouldn't hurt to have the binary.

File: mingw-w64-capstone/PKGBUILD [hunk 1]
```diff
@@ -10,7 +10,7 @@ pkgrel=3
 pkgdesc='Lightweight multi-platform, multi-architecture disassembly framework'
 url='https://www.capstone-engine.org/index.html'
 arch=('any')
-mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32', 'clangarm64')
```
Comment (jeremyd2019) [inline mingw-w64-capstone/PKGBUILD:13]:
```suggestion
mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
```

No commas here

**Comment by jeremyd2019** [comment]
https://github.com/msys2-arm/MINGW-packages/actions/runs/3895710764/jobs/6651362448 - worked

File: mingw-w64-capstone/PKGBUILD (hunk 1 above)
Comment (heljkon) [reply mingw-w64-capstone/PKGBUILD]: awkward fault, thanks

**Comment by heljkon** [comment]
> I thought about having qemu on arm64, but AFAIK whpx is part of the x86 section of qemu code, so probably wouldn't _just work_ on Windows on ARM64. If the only option is tcg, may as well run tcg on an x86_64 box that is much more common.

I'd agree if accelerated virtualization was the only application. QEMU contains other tools as well, which may be interesting on Windows on ARM64, too. QEMU Agent, Qemu Image and of course emulation of other systems.

**Pull request merged**